<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionDesProduits</class>
 <widget class="QDialog" name="GestionDesProduits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion Des Produits</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QGroupBox" name="BG">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>700</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="IMG">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>700</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="pixmap">
     <pixmap>FF.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="G4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>113</width>
      <height>76</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(39, 39, 39);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QComboBox" name="comboBox_02">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>60</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Supprimer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>38</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="G1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>362</width>
      <height>182</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(39, 39, 39);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_01">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_03">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>66</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_02">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>38</y>
       <width>27</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_05">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>122</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Qte</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_02">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>38</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_06">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>150</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_06">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>22</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Prix</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="Ajouter">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_01">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>10</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_04">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>94</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_03">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>66</y>
       <width>92</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Date Production</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_04">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>94</y>
       <width>88</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Date Expiration</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_05">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>122</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="Inserer_IMG">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>100</y>
       <width>147</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Inserer Image Prouit</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Statistique">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>620</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(39, 39, 39);</string>
    </property>
    <property name="text">
     <string>Statistique</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="G2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>1003</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(39, 39, 39);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="PDF">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>100</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Trier">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>20</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Trier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Chercher">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>60</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Chercher</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="TAB_P">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>891</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Quitter">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>654</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(39, 39, 39);</string>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="G3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>213</width>
      <height>182</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(39, 39, 39);</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="lineEdit_08">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>66</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>122</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_08">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>38</y>
       <width>27</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_09">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>66</y>
       <width>92</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Date Production</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_07">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_09">
     <property name="geometry">
      <rect>
       <x>114</x>
       <y>94</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>22</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Prix</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_07">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>38</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>122</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Qte</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>94</y>
       <width>88</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Date Expiration</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="Modifier">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>133</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_01">
     <property name="geometry">
      <rect>
       <x>43</x>
       <y>10</y>
       <width>60</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_11">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>150</y>
       <width>50</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <zorder>IMG</zorder>
   <zorder>G4</zorder>
   <zorder>G1</zorder>
   <zorder>G3</zorder>
   <zorder>G2</zorder>
   <zorder>Quitter</zorder>
   <zorder>Statistique</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
